eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
+
+type RouteList []*Route
```

Comment:
when reading other go code, I tend to prefer not having empty type aliases like this. It makes sense when it has its own methods appended, or just a number type gets a name for some constants or binary flags, but for such cases recently started preferring the non-aliased format. A slice of routes is just that: a slice of routes. What do you think, would it make sense to remove this alias? As I checked, we don't have these now in the public interfaces of the packages, and I remember removing a lot of my own ones during an earlier refactoring.